src/Dfe.EarlyYearsQualification.Content/RichTextParsing/IGovUkContentfulParser.cs

```diff
@@ -0,0 +1,8 @@
+using Contentful.Core.Models;
+
+namespace Dfe.EarlyYearsQualification.Content.RichTextParsing;
+
+public interface IGovUkContentfulParser
```

Comment:
Interfaces are usually implementation agnostic so this might be better named something like IGovUkContentParser with the implementation being named GovUkContentfulParser. That being said, it is tightly coupled to Contentful given the Document object so may be alright in this instance.